shopinvader_restapi/models/shopinvader_backend.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2020 Camptocamp (http://www.camptocamp.com).
+# @author Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import hashlib
+import os
+from contextlib import contextmanager
+
+from odoo import _, api, fields, models, tools
+from odoo.osv import expression
+
+from odoo.addons.base_sparse_field.models.fields import Serialized
+
+
+class ShopinvaderBackend(models.Model):
+    _name = "shopinvader.backend"
+    _inherit = [
+        "collection.base",
+        "server.env.techname.mixin",
+        "server.env.mixin",
+    ]
+    _description = "Shopinvader Backend"
+
+    name = fields.Char(required=True)
+    company_id = fields.Many2one(
+        "res.company",
+        "Company",
+        required=True,
+        default=lambda s: s._default_company_id(),
+    )
+    notification_ids = fields.One2many(
+        "shopinvader.notification",
+        "backend_id",
+        "Notification",
+        help="Send mail for predefined events",
+    )
+    nbr_cart = fields.Integer(compute="_compute_nbr_sale_order")
+    nbr_sale = fields.Integer(compute="_compute_nbr_sale_order")
+    allowed_country_ids = fields.Many2many(
+        comodel_name="res.country", string="Allowed Country"
+    )
+    anonymous_partner_id = fields.Many2one(
+        "res.partner",
+        "Anonymous Partner",
+        help=("Provide partner settings for unlogged users " "(i.e. fiscal position)"),
+        required=True,
+        default=lambda self: self.env.ref("shopinvader_restapi.anonymous"),
+    )
+    sequence_id = fields.Many2one(
+        "ir.sequence", "Sequence", help="Naming policy for orders and carts"
+    )
+    lang_ids = fields.Many2many("res.lang", string="Lang", required=True)
```

Comment:
```suggestion
    lang_ids = fields.Many2many("res.lang", string="Lang")
```

There is no need anymore to force to specify the lang.